### PR TITLE
Dev1 - Add list api cartItem and func send response data

### PR DESCRIPTION
**Pull request by congtran8991**
- Add list api cartItem and func send response data

### CODE REVIEW

File: ecommerge_server/src/api/cart-item/controllers/cart-item.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        }
+        const entries = await strapi.db
+          .query("api::cart-item.cart-item")
+          .findMany({
+            populate: arrFieldPopulate,
+            where: query,
+          });
+        helper.sendSuccess(ctx, message.success, entries);
+      } catch (error) {
+        helper.sendFailed(ctx, message.success);
```
Comment (nguyenhoaidanh) [inline ecommerge_server/src/api/cart-item/controllers/cart-item.js:36]: sendFailed vs message.success

File: ecommerge_server/src/api/cart-item/controllers/cart-item.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        select: selectField.arrProduct,
+      };
+      const selectUsers = {
+        select: selectField.arrUsers,
+      };
+      try {
+        const entries = await strapi.db
+          .query("api::cart-item.cart-item")
+          .update({
+            where: { id: 1 },
```
Comment (nguyenhoaidanh) [inline ecommerge_server/src/api/cart-item/controllers/cart-item.js:70]: id hardcode

File: ecommerge_server/src/api/cart-item/controllers/cart-item.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      }
+    },
+    async create(ctx) {
+      const { amount = 1, idProduct, idUsers } = ctx.request.body;
+      try {
+        if ((idProduct, idUsers)) {
+          await strapi.db.query("api::cart-item.cart-item").create({
+            data: {
+              amount: amount,
+              product: idProduct, // id của product
```
Comment (nguyenhoaidanh) [inline ecommerge_server/src/api/cart-item/controllers/cart-item.js:90]: no need to add comment

File: ecommerge_server/src/utils/server.js [hunk 4]
```diff
@@ -18,9 +18,29 @@ const isValidPassWord = (str) => {
   return str.length > 7 && str.length < 20;
 };
 
+const sendSuccess = (ctx, message, data = []) => {
+  return ctx.send({
+    code: 200,
+    message: message,
+    data,
+    status: true
```
Comment (nguyenhoaidanh) [inline ecommerge_server/src/utils/server.js:26]: what is `status` used for?

File: ecommerge_server/src/api/cart-item/controllers/cart-item.js (hunk 1 above)
Comment (congtran8991) [reply ecommerge_server/src/api/cart-item/controllers/cart-item.js]: done

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**